pagopa/io-app
# feat: [IOBP-311,IOBP-317] New wallet payment method selection screen

## Short description
added said screen 
<img width="200" alt="Screenshot 2023-12-18 at 15 17 59" src="https://github.com/pagopa/io-app/assets/60693085/1924dc00-4ed0-4a58-a78b-e33a2222d9a1">

## List of changes proposed in this pull request
- added screen
- added required i18n keys
- added required selectors

## How to test
with the dev-server active, navigate to the profile section, then to "new Wallet playground" and make sure the second screen of the payment flow correctly follows the design.
also, if you keep going through the flow you should be able to see the data of the payment method you picked two pages lates, under the "selectedMethodOption" section.
also, the screen currently does NOT handle the selection of non-saved methods.
